tsconfig.json

```diff
@@ -39,7 +39,8 @@
         "ReactTypings",
         "tools/",
         "test/resources/sampleReactNative022Project/",
-        "test/smoke/"
+        "test/smoke/",
+        "src/debugger/direct"

```

Comment:
Yes, it is. I temporarily excluded direct debugger, since we are going to get rid off vscode-chrome-debug-core lib, but for now RN Hermes engine doesn't support flat sessions API, so we cannot debug RN Hermes applications.